test/fixtures/test-runner/output/test-timeout-flag.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --test-timeout=500
```

Comment:
500 seems a bit too much, don't you think? Also, could you please format this file using the same rules we have on other area of the repo?
```suggestion
// Flags: --test-timeout=20
```

Reply:
Thanks for the suggestion! Can I ask is there an easy way to format this file?

Reply:
I just accidentally reverted the `semi-column` fix, will push again after properly format the file

Reply:
Thanks for the suggestion for the `timeout` value, maybe I should also set the second timeout to be lower (instead of 600ms)?

Reply:
Gotach! I tried to comment out `!test/fixtures/**/.*` (on line 12) in `.gitignore` but `make lint-js-fix` didn't help, wasn't sure if I did something incorrectly. But move file outside of the fixtures dir worked 👍 thanks again for the useful tips.

I have dropped the timeout to 20ms and improved the test descriptions.